webapp/components/NewsForm/NewsForm.spec.js

```diff
@@ -1,32 +1,49 @@
 import { mount } from '@vue/test-utils'
 import NewsForm from '@/components/NewsForm/NewsForm.vue'
 
+const mutate = jest.fn().mockResolvedValue({ write: { id: 20, title: 'Item 1', votes: 0 } })
+
 describe('Tests for NewsForm.vue', () => {
+  let wrapper
+
+  const createComponent = (data) => {
+    wrapper = mount(NewsForm, {
+      mocks: {
+        $apollo: {
+          mutate
+        },
+        $apolloHelpers: {
+          getToken: jest.fn().mockResolvedValue(undefined)
+        }
+      },
+      data () {
+        return {
+          newsTitle: data
+        }
+      }
+    })
+  }
+
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
   describe(':buttons', () => {
     describe('click create', () => {
-      let newsForm
-      beforeEach(() => {
-        newsForm = mount(NewsForm, {
-          data () {
-            return {
-              newsTitle: ''
-            }
-          }
-        })
-      })
       describe('when input is empty', () => {
         it('is is disabled', () => {
-          const createButton = newsForm.find('button.create-item-button')
+          createComponent('')
+
+          const createButton = wrapper.find('button.create-item-button')
           expect(createButton.attributes('disabled')).toBe('disabled')
         })
       })
 
       describe('when input is not empty', () => {
-        beforeEach(async () => {
-          await newsForm.find('input').setValue('Title example')
-        })
-        it('is not disabled', () => {
-          const createButton = newsForm.find('button.create-item-button')
+        it('is not disabled', async () => {
+          createComponent('')
+          await wrapper.find('input').setValue('Title example')
+          const createButton = wrapper.find('button.create-item-button')
           expect(createButton.attributes('disabled')).toBe(undefined)
         })
       })
```

Comment:
⭐ For your previous frontend tests still passing. Requests to the backend are mocked.